src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER_OF_ELEMENTS = 10;
+    private K[] key;
+    private V[] value;
+    private int size = 0;
```

Comment:
int type = 0 by default
```suggestion
    private int size;
```